packages/bot-web-ui/src/components/summary/summary-card.jsx

```diff
@@ -95,7 +114,7 @@ const SummaryCard = ({
 };
 
 SummaryCard.propTypes = {
-    contract: PropTypes.object,
+    contract_info: PropTypes.object,
```

Comment:
Could you also add types of other added props here?

Reply:
Ah, missed them. Thank you 👍